# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/atracdenc/hotfix-inttypes.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- atracdenc-0.1.1/src/bitstream/bitstream.h.vanilla	2025-02-22 12:15:41.949948411 +0100
+++ atracdenc-0.1.1/src/bitstream/bitstream.h	2025-02-22 12:15:53.526614856 +0100
@@ -18,7 +18,7 @@
 
 #pragma once
 #include <vector>
-
+#include <cinttypes>
 #include <iostream>
 
 namespace NBitStream {
